doc/configuration.rst

```diff
@@ -61,18 +61,26 @@ And some things can be tweaked directly in CSS:
 
 - ``.sphx-glr-thumbcontainer`` (:ref:`setting_thumbnail_size`)
 
-You can also use the package ``warnings`` in the ``conf.py`` file to prevent
-warnings from being captured and included in your built documentation. For
-example, to remove the specific Matplotlib agg warning, you can add::
+.. _removing_warnings:
+
+Removing warnings
+=================
+
+To prevent warnings from being captured and included in your built
+documentation, you can use the package ``warnings`` in the ``conf.py`` file.
+For example, to remove the specific ``UserWarning``: 'I don't want this
+warning', you can add::
 
     import warnings
 
     warnings.filterwarnings("ignore", category=UserWarning,
-                            message='Matplotlib is currently using agg, which is a'
-                                    ' non-GUI backend, so cannot show the figure.')
+                            message='I don't want this warning.')
```

Comment:
This is less clear than before -- before it was clear that the `message` arg is somehow the thing you want to suppress, and it was a good example of one that you'd want to. I would either leave this, and say below that this is actually one that we already remove, or choose a more suitable `message` that people will recognize / have probably seen.

Reply:
Good point, I wasn't sure about that as well. Amended back.